shaman/models/projects.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm.exc import DetachedInstanceError
+from shaman.models import Base
+
+
+class Project(Base):
+
+    __tablename__ = 'projects'
+    id = Column(Integer, primary_key=True)
+    name = Column(String(256), nullable=False, unique=True, index=True)
+
+    def __init__(self, name):
+        self.name = name
+
+    def __repr__(self):
+        try:
+            return '<Project %r>' % self.name
+        except DetachedInstanceError:
+            return '<Project detached>'
+
+    def __json__(self):
+        json_ = {}
+        json_["name"] = self.name
+        return json_
+
+
+def get_or_create(name, **kw):
```

Comment:
We don't, I think this was originally pulled from https://github.com/ceph/shaman/blob/master/shaman/models/__init__.py#L62 where ``**kw`` does make sense.